views/index.ejs

```diff
@@ -11,7 +11,9 @@
     <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
```

Comment:
Since all the items are in an array, how can we find the # of things inside said array? Hint: length!